Jenkinsfile

```diff
@@ -22,14 +22,15 @@ node {
             def imageTag = "${DOCKER_ACCOUNT}/reshuffle:${BRANCH_NAME}";
             stage('Build') {
                 echo 'Building docker image';
-                sh "sudo docker build -f testing.Dockerfile ${NO_CACHE ? '--no-cache' : ''} --tag ${imageTag} .";
+                sh "sudo docker build -f testing.Dockerfile ${params.NO_CACHE ? '--no-cache' : ''} --tag ${imageTag} .";
             }
             stage('Push') {
                 echo 'Pushing docker image';
                 sh 'sudo $(aws ecr get-login --no-include-email --region us-east-1)';
                 sh "sudo docker push ${imageTag}";
+                sh "sudo docker tag ${imageTag} binaris/reshuffle:${BRANCH_NAME}";
```

Comment:
Maybe `binaris/reshuffle-integration`?

Reply:
It's not an image specifically for integration, just a built bundle.